src/panels/lovelace/views/hui-view.ts

```diff
@@ -5,6 +5,7 @@ import {
   TemplateResult,
   property,
 } from "lit-element";
+import Muuri from "muuri/dist/muuri";
```

Comment:
We should only import it in edit mode

Reply:
What is the best way to do this and where? Dynamic imports are async. so not sure if we put the import in `updated` and then update ifs to `if imported and in edit mode`

Reply:
Add it to <https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/panels/lovelace/views/hui-view-editable.ts>